src/Razor/src/Microsoft.AspNetCore.Razor.Common/Telemetry/ITelemetryReporter.cs

```diff
@@ -1,13 +1,15 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT license. See License.txt in the project root for license information.
 
+using System;
 using System.Collections.Immutable;
 using Microsoft.VisualStudio.Telemetry;
 
 namespace Microsoft.AspNetCore.Razor.Telemetry;
 
-internal interface ITelemetryReporter
+public interface ITelemetryReporter
```

Comment:
because of

 `public LoggerAdapter(ILogger logger, ITelemetryReporter? telemetryReporter)`

If not made public, `LoggerAdapter` it would end up with inconsistent accessibility because `LoggerAdapter` is public